bundler.d/libvirt.rb

```diff
@@ -1,4 +1,4 @@
 group :libvirt do
-  gem 'fog-libvirt', '>= 0.9.0'
+  gem 'fog-libvirt', ' 0.12.0'
```

Comment:
I don't think we should pin an exact version. How about 

```suggestion
  gem 'fog-libvirt', '>= 0.9.0', '!= 0.12.1'
```

Reply:
well I guess we could do that

Reply:
Are we certain a 0.12.2 won't break the pipeline again?

Reply:
Depends on @ekohl ;-)

Reply:
I didn't intend to break it in the first place ;)